### PR TITLE
add tag filtering

### PR DESCRIPTION
**Pull request by codiebeulaine**
## Purpose
_Need to be able to filter pages via tag_

### CODE REVIEW

File: home/api.py [hunk 1]
```diff
@@ -3,19 +3,29 @@
...
+    ])
+
+    def get_queryset(self):
+        queryset = super(ContentPagesViewSet, self).get_queryset()
+        tag = self.request.query_params.get('tag')
+        if tag is not None:
+            ids = []
+            for t in ContentPageTag.objects.all():
+                if t.tag.name == tag:
+                    ids.append(t.content_object_id)
```
Comment (KaitCrawford) [inline home/api.py:22]: Is this a list of Tag objects?
Is that what we're returning or are we returning content pages that have these tags?

Comment (codiebeulaine) [reply home/api.py:22]: we are returning the content pages. so we collect the IDs fo the pages that match the tag then filter them against the queryset given by super

Comment (KaitCrawford) [reply home/api.py:22]: ah I see. `t.content_object_id` is what I missed. so this is a list of content page ids

**Review by KaitCrawford — APPROVED** [review]
:+1:

**Pull request merged**